tests/test_pyfunction.rs

```diff
@@ -114,23 +113,20 @@ fn test_functions_with_function_args() {
     });
 }
 
-#[cfg(not(Py_LIMITED_API))]
```

Comment:
Not directly related to this PR; just noticed that these attributes are probably not necessary so deleted them.